tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/device_info.py

```diff
@@ -35,8 +48,36 @@ def tap_more_info(self):
         self.marionette.find_element(*self._more_info_button_locator).tap()
         return self.MoreInfo(self.marionette)
 
+    # In order to access UI, the frame needs to be switched to root
+    def tap_update_frequency(self):
+        self.marionette.find_element(*self._update_frequency_locator).tap()
+        self.device_info_frame = self.marionette.get_active_frame()
+        self.marionette.switch_to_frame()
+        Wait(self.marionette).until(expected.element_displayed(
+            self.marionette.find_element(*self._update_ok_button_locator)))
+
+    # When leaving the page, return to the saved application frame
+    def exit_update_frequency(self):
+        self.marionette.find_element(*self._update_ok_button_locator).tap()
+        self.marionette.switch_to_frame(self.device_info_frame)
```

Comment:
Last time I used it it was somewhat unreliable. I'll try it, and if it works, I'll change to that.